TRAVIS_WEBHOOK_README.md

```diff
@@ -1,18 +1,14 @@
 # Travis Webhook Instructions
 
-## Make sure travis is installed with gem
+## Make sure travis is installed with Homebrew
 
-`gem install travis`
+`brew install travis`
 
 ## Login to travis
 
-`travis login --pro`
+Create a personal access token on github.
```

Comment:
Nitpick: `GitHub`

Reply:
Maybe adding a link to generating a token would help: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token